### ISSUE TITLE
Indicate which bug would be the best to start working on (qx.html)

### ISSUE THREAD

**Issue by designakt** [bwinton/d3Experiments#2]
Assuming qx.html could be of value to people who want to pick a bug to start working on Firefox, I assume they would like to see which is the most important bug to start working on.
Sort by priority does this, sort of. Can we, in addition to priority show the number of votes? And sort by default by number of votes instead of bug#, so that the top bugs are those that are not assigned, have high priority, and have the most votes?

(This might also help us internally better see which bugs are important but might miss spec, mentor or link. _For example: bug 260611: says it misses spec, but the bug looked to me as if it is clear what there is to do..._ )

**Comment by bwinton** [comment]
No.  Votes are mostly ignored by developers, and (in my opinion) rightly so.  They're extremely vulnerable to groups of people getting together and pushing their pet bug to the top, regardless of what is actually important to the majority of our users.

I would be in favour of making the default view sort-by-priority, though.  :smile:

And if you feel there's a spec for bug 260611, please feel free to change the whiteboard to [qx:link:spec], and the dashboard will automatically update.  (I'm trying to not be the only person to fill in those fields, since then everyone would be waiting for me while I'm off doing other stuff.  :wink:)
